C/Cairo/build_tarballs.jl

```diff
@@ -7,18 +7,12 @@ version = v"1.14.12"
 sources = [
     "https://www.cairographics.org/releases/cairo-$(version).tar.xz" =>
     "8c90f00c500b2299c0a323dd9beead2a00353752b2092ead558139bd67f7bf16",
-    "./bundled"
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/cairo-*/
 
-export LDFLAGS="${LDFLAGS} -L${libdir}"
-# TODO: find out why we need to manually add "${prefix}/include/freetype2",
-# the pkg-config file for freetype2 should already have this information.
-export CPPFLAGS="-I${prefix}/include -I${prefix}/include/freetype2"
```

Comment:
Hmmm, looks like `cairo` still can't find `zlib` without `-I${prefix}/include`, because it doesn't have the proper `.pc` file.

Reply:
Wait, it worked for me. Which platform? 😕